pkgs/by-name/jp/jpilot/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  gtk3,
+  intltool,
+  libgcrypt,
+  pilot-link,
+  pkg-config,
+  sqlite,
+}:
+
+stdenv.mkDerivation {
+  pname = "jpilot";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "juddmon";
+    repo = "jpilot";
+    rev = "v2_0_2";
```

```suggestion
    rev = "v${lib.replaceStrings [ "." ] [ "_" ] version}";
```

Add `rec` to `stdenv.mkDerivation`